cmd/lora-app-server/main.go

```diff
@@ -600,6 +600,16 @@ func main() {
 			Usage:  "mqtt CA certificate file used by the gateway backend (optional)",
 			EnvVar: "MQTT_CA_CERT",
 		},
+		cli.StringFlag{
```

Comment:
Naming these `mqtt-tls-cert` and `mqtt-tls-key` is consistent with the other config options. Could you rename these?